cmd/fakeit/run.go

```diff
@@ -29,18 +27,13 @@ func (r *run) runE(cmd *cobra.Command, args []string) error {
 
 	// --------------------------------------------------------------------- //
 
-	var srv *server.Server
-	var wrk *worker.Worker
+	var dae *daemon.Daemon
 	{
-		srv, wrk, err = daemon.Create(env)
-		if err != nil {
-			return tracer.Mask(err)
-		}
+		dae = daemon.Create(env)
 	}
 
 	{
-		go srv.Daemon()
-		go wrk.Daemon()
+		go dae.Server().Daemon()
 	}
```

Comment:
For the `apiserver fakeit` command we run only run the server now. The worker is not needed to generate some fake data. We only need the server in order to get "remote" access to Redis.